pep-0001.txt

```diff
@@ -486,9 +486,8 @@ Each PEP should have the following parts/sections:
 
 12. References -- A collection of URLs used as references through the PEP.
 
-13. Copyright/public domain -- Each PEP must either be explicitly
-    labeled as placed in the public domain (see this PEP as an
-    example) or licensed under the `Open Publication License`_.
+13. Copyright/license -- Each new PEP must be placed under a dual license of
+    public domain and CC0-1.0-Universal_ (see this PEP for an example).
```

Comment:
Does it have to be both, or can one or the other suffice?  I don't have @VanL 's decision on this handy.

Reply:
This is coming directly from @VanL in the issue that is referenced in the PR.

Reply:
It's a general dual licensing thing: as a publisher you license under both, and then as a consumer you can accept it under either.